languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/grammar.xml

```diff
@@ -10069,6 +10069,17 @@ though, and has since been slightly modified)-->
     </rulegroup>
   </category>
   <category id="CAT_GRAMMAIRE" name="Grammaire">
+      <rule id="IL_FAUT" name="Il faut">
+          <antipattern>
+              <token postag="M fin inte"></token>
+          </antipattern>
+          <pattern>
+              <token postag="SENT_START"/>
+              <marker>     <token>Faut</token></marker>
```

Comment:
Please put marker and tokens in separate lines:

```xml
<marker>
  <token>faut</token>
</marker>
```

Also: no need to write a capital "F" — unless you make the token `case_sensitive="yes"`, `faut` will find both `Faut` and `faut` (and `FAUT`, `faUt` etc.).